kyaml/Makefile

```diff
@@ -36,6 +36,9 @@ vet:
 	go vet ./...
 
 openapi:
+	sudo apt-get install jq
```

Comment:
OK, we cannot do sudo in a makefile

Time to peel this out to shell script.  Suggest
```
openapi:
   ./hack/fetchOpenApiData.sh
```
and at the top of that script use something like
```
if ! command -v jq &> /dev/null ; then
    echo Please install jq
    echo on ubuntu: sudo apt-get install jq
    exit 1
fi
```
i.e. leave it up to the user to install it out of band (for now).

I don't favor attempting to download and install to `MYGOBIN`, as with `kind` and `helm`, as this is a more generic (non-kubernetes) tool.